WordPress/Classes/ViewRelated/Menus/MenusViewController.m

```diff
@@ -728,7 +728,7 @@ - (void)itemsViewController:(MenuItemsViewController *)itemsViewController prefe
 - (void)itemsViewController:(MenuItemsViewController *)itemsViewController prefersAdjustingScrollingOffsetForAnimatingView:(UIView *)view
 {
     // adjust the scrollView offset to ensure this view is easily viewable
-    const CGFloat padding = 10.0;
+    const CGFloat scrollingOffsetAdjustmentPadding = 10.0;
```

Comment:
Could we move the const declarations to the top of the file, outside the implementation and capitalize the first letter for consistency with other instances in the app.  For example, BlogService.m, or PostService.m